src/chocolatey/infrastructure.app/services/ChocolateyPackageInformationService.cs

```diff
@@ -268,5 +270,51 @@ public void save_package_information(ChocolateyPackageInformation packageInforma
         public void remove_package_information(IPackageMetadata package)
             => Remove(package);
 #pragma warning restore IDE1006
+
+        private static string GetStorePath(IFileSystem fileSystem, string id, NuGetVersion version)

```

Comment:
I think we should put this above the deprecated methods, just for consistency's sake, pretty much everywhere the deprecated methods are the last thing in their files.

Reply:
Good thinking. I've moved this above the deprecations.